### PR TITLE
I told you I'd do it - Changes Batong and Supermatter toy descriptions

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

image

## Why It's Good For The Game

Holy shit please make it stop please make it st

## Changelog
:cl:
spellcheck: Literally fixes the toy Batong.
/:cl:

### CODE REVIEW

File: fulp_modules/toys/code/toys.dm [hunk 1]
```diff
@@ -1,26 +1,26 @@
 /obj/item/toy/plush/batong
 	name = "batong"
-	desc = "A cheaply made toy. Looks like it need some recharge maybe security can help you"
+	desc = "A cheaply made toy resembling a Stunbaton."
```
Comment (SgtHunk) [inline fulp_modules/toys/code/toys.dm:3]:
```suggestion
	desc = "A cheaply made toy resembling a stun baton."
```

**Review by SgtHunk — CHANGES_REQUESTED** [review]

**Comment by SgtHunk** [comment]
Soulless

File: fulp_modules/toys/code/toys.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply fulp_modules/toys/code/toys.dm:3]: I hope you rot in the depths of hell

Comment (SgtHunk) [reply fulp_modules/toys/code/toys.dm]: :trollface:

**Comment by Helianthus00** [comment]
no soul

**Pull request closed without merging**